### ISSUE TITLE
Motion-Alarm with no motion

### ISSUE THREAD

**Issue by dirkg173** [mruiter/com.neo#182]
For no reason, 1 of 3 motion sensor suddenly reports motion even though there is no motion. After the set time, the motion alarm goes off and immediately on again.
I have not made any changes in the Z-Wave area. A Homey restart (no PTP) did not help.
The sensor is already several months in use and I had no problems so far.

Diagnostic report no.: 409d4491-1c48-4d64-bd1a-73a4ec788628

![E25C7133-7502-4387-8081-DE915F04DB9B](https://user-images.githubusercontent.com/40319629/107886917-0a1a1a80-6f03-11eb-9d35-5ad36e8b21a8.jpeg)
![17B9415D-E8D7-4FFB-A596-8D79445AE49A](https://user-images.githubusercontent.com/40319629/107886918-0be3de00-6f03-11eb-9840-151838693a54.png)

**Comment by mruiter** [comment]
Appversion and homey firmware version ?

Verstuurd vanaf mijn iPhone

**Comment by dirkg173** [comment]
I thought the Homey firmware and app version are in the diagnostic report.
Homey FW: v5.0.0
App version: v2.1.10 Experimental

Probably I found the problem. When the problems occurred, the battery capacity was displayed at 33%. A few hours later, starting around 22:20, the battery capacity dropped to 1%.
I replaced the battery this morning and so far no more error messages have appeared.

<img width="965" alt="Bildschirmfoto 2021-02-15 um 10 40 16" src="https://user-images.githubusercontent.com/40319629/107930215-fad6b380-6f7a-11eb-89e4-0293614d99fe.png">

Sorry for the inconvenience.
